web/client/plugins/MeasureResults.jsx

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Can we have just one plugin that uses different components for the Toolbar and DrawerMenu?